lib/facter/swap_file_size.rb

```diff
@@ -0,0 +1,19 @@
+Facter.add('swap_file_size') do
```

Comment:
I just saw #47, which appears to return a hash which requires `stringify` to be disabled, which may not work for many people who are still running PE 3.8 :(